Source/cmake/OptionsGTK.cmake

```diff
@@ -45,12 +45,6 @@ if (WTF_CPU_ARM OR WTF_CPU_MIPS)
     SET_AND_EXPOSE_TO_BUILD(USE_CAPSTONE ${DEVELOPER_MODE})
 endif ()
 
-# For old versions of HarfBuzz that do not expose an API for the OpenType MATH
-# table, we enable our own code to parse that table.
-if ("${PC_HARFBUZZ_VERSION}" VERSION_LESS "1.3.3")
-    add_definitions(-DENABLE_OPENTYPE_MATH=1)
-endif ()
```

Comment:
This means we must ensure `OPENTYPE_MATH` is always disabled. See PlatformEnable.h:

```
#if !defined(ENABLE_OPENTYPE_MATH) && (OS(DARWIN) && USE(CG)) || (USE(FREETYPE) && !PLATFORM(GTK)) || (PLATFORM(WIN) && (USE(CG) || USE(CAIRO)))
#define ENABLE_OPENTYPE_MATH 1
#endif
```
That means we would enable it for WPE, so I think we should remove the `(USE(FREETYPE) && !PLATFORM(GTK))`

We should also change `#if USE(HARFBUZZ) && !ENABLE(OPENTYPE_MATH)` with `#if USE(HARFBUZZ)`

Reply:
I've seen PR 10461 now, we still need to change the #ifdefs, though

Reply:
Do you want me to remove these guards too?

```
WebCore/platform/graphics/freetype/FontCacheFreeType.cpp:#if ENABLE(VARIATION_FONTS)
WebCore/platform/graphics/freetype/FontCacheFreeType.cpp:#if ENABLE(VARIATION_FONTS)
WebCore/platform/graphics/freetype/FontCacheFreeType.cpp:#if ENABLE(VARIATION_FONTS)
WebCore/platform/graphics/freetype/FontCacheFreeType.cpp:#if ENABLE(VARIATION_FONTS)
WebCore/platform/graphics/freetype/FontCacheFreeType.cpp:#endif // ENABLE(VARIATION_FONTS)
WebCore/platform/graphics/freetype/FontCacheFreeType.h:#if ENABLE(VARIATION_FONTS)
WebCore/platform/graphics/freetype/FontCustomPlatformDataFreeType.cpp:#if ENABLE(VARIATION_FONTS)
WebCore/platform/graphics/freetype/FontCustomPlatformDataFreeType.cpp:#if ENABLE(VARIATION_FONTS)
WebCore/platform/graphics/freetype/FontCustomPlatformDataFreeType.cpp:#if ENABLE(VARIATION_FONTS)
WebCore/platform/graphics/freetype/FontPlatformDataFreeType.cpp:#if ENABLE(VARIATION_FONTS)
WebCore/platform/graphics/freetype/FontPlatformDataFreeType.cpp:#if ENABLE(VARIATION_FONTS)
WebCore/platform/graphics/harfbuzz/ComplexTextControllerHarfBuzz.cpp:#if ENABLE(VARIATION_FONTS)
WebCore/platform/graphics/harfbuzz/ComplexTextControllerHarfBuzz.cpp:#if ENABLE(VARIATION_FONTS)
```

I have left them for now.

Reply:
I guess we could remove them, I think we have other cases of ifdefs or feature we unconditionally enable, so it's up to you.